data-plane/benchmarks/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/filter/SuffixFilterBenchmark.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.knative.eventing.kafka.broker.dispatcher.impl.filter;
+
+import dev.knative.eventing.kafka.broker.dispatcher.Filter;
+import dev.knative.eventing.kafka.broker.dispatcher.impl.filter.subscriptionsapi.PrefixFilter;
+import io.cloudevents.CloudEvent;
+import io.cloudevents.core.builder.CloudEventBuilder;
+import io.cloudevents.core.v1.CloudEventV1;
+import java.net.URI;
+import java.util.Map;
+
+public class SuffixFilterBenchmark {
+    public static CloudEvent event() {
+        return CloudEventBuilder.v1()
+                .withId("abcdefghijklmnop")
+                .withSource(URI.create("http://localhost"))
+                .withType("com.github.pull.create")
+                .withDataSchema(URI.create("/api/schema"))
+                .withDataContentType("testContentType")
+                .withSubject("testSubject")
+                .build();
+    }
+
+    public static class SuffixFilterIDBenchmark extends FilterBenchmark {
+
+        @Override
+        protected Filter createFilter() {
+            return new PrefixFilter(Map.of(CloudEventV1.ID, "lmnop"));
```

Comment:
Just curious if the prefix that need to be matched is really long, will that have impact on the benchmark result? @Cali0707

Reply:
And why it is `PrefixFilter` here?